src/getActors.js

```diff
@@ -0,0 +1,23 @@
+const fetch = require('node-fetch');
+
```

Comment:
I'm not sure what this function does. It looks like it calls an API to get people? Consider renaming the function to something like ```function getPeople(id)```